cluster/gce/gci/configure-helper.sh

```diff
@@ -2286,6 +2286,23 @@ EOF
   fi
 }
 
+# A helper function to set up a custom yaml for a k8s addon.
+#
+# $1: addon category under /etc/kubernetes
+# $2: manifest source dir
+# $2: manifest file
+# $3: custom yaml
+function setup-addon-custom-yaml {
+  local -r manifest_path="/etc/kubernetes/$1/$2/$3"
+  local -r custom_yaml="$4"
+  if [ -n "${custom_yaml:-}" ]; then
+    # Replace with custom manifest.
+    cat > "${manifest_path}" <<EOF
+$(echo "${custom_yaml}")
```

Comment:
Why is this execing echo?  Why not simply reference the variable?

Reply:
It does not need echo cmd and verified it. I just copied line 2261 $(echo "$CUSTOM_KUBE_DNS_YAML"). I will send another PR to clean it up.